code/modules/food_and_drinks/food.dm

```diff
@@ -12,3 +12,17 @@
 		..()
 		pixel_x = rand(-5, 5)	//Randomizes postion slightly.
 		pixel_y = rand(-5, 5)
+
+
+/obj/item/weapon/reagent_containers/food/proc/checkLiked(var/fraction, mob/M)
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		if(foodtype & H.dna.species.toxic_food)
+			to_chat(H,"<span class='warning'>What the hell was that thing?!</span>")
+			H.adjust_disgust(25 + 30 * fraction)
+		else if(foodtype & H.dna.species.disliked_food)
+			to_chat(H,"<span class='notice'>That didn't taste very good...</span>")
+			H.adjust_disgust(11 + 15 * fraction)
+		else if(foodtype & H.dna.species.liked_food)
+			to_chat(H,"<span class='notice'>I love this taste!</span>")
```

Comment:
when you bite something for the first time e.g. a monkey meat steak 🍔
it will tell you what you taste and obviously you will taste monkey meat and bun/bread
make it appear everytime it shows what you taste